<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Kanban Test" duration-ms="11925" started-at="2015-10-21T15:13:17Z" finished-at="2015-10-21T15:13:29Z">
    <groups>
    </groups>
    <test name="Sanity test" duration-ms="11925" started-at="2015-10-21T15:13:17Z" finished-at="2015-10-21T15:13:29Z">
      <class name="com.suvi.tests.BaseTest">
        <test-method status="PASS" signature="initializeTestBaseSetup(java.lang.String, java.lang.String)" name="initializeTestBaseSetup" is-config="true" duration-ms="11824" started-at="2015-10-21T15:13:17Z" finished-at="2015-10-21T15:13:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://kanbanflow.com/]]>
              </value>
            </param>
          </params>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-10-21T15:13:29Z" finished-at="2015-10-21T15:13:29Z">
        </test-method>
        <test-method status="FAIL" signature="verifyHomePage()" name="verifyHomePage" duration-ms="16" started-at="2015-10-21T15:13:29Z" finished-at="2015-10-21T15:13:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Home page title doesn't match expected:<true> but was:<false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Home page title doesn't match expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at com.suvi.tests.BaseTest.verifyHomePage(BaseTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:203)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="tearDown()" name="tearDown" is-config="true" duration-ms="74" started-at="2015-10-21T15:13:29Z" finished-at="2015-10-21T15:13:29Z">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
